wolfcrypt/src/port/kcapi/kcapi_ecc.c

```diff
@@ -117,15 +118,21 @@ int KcapiEcc_MakeKey(ecc_key* key, int keysize, int curve_id)
         ret = kcapi_kpp_ecdh_setcurve(key->handle, kcapiCurveId);
     }
     if (ret == 0) {
-        ret = (int)kcapi_kpp_keygen(key->handle, key->pubkey_raw,
+        ret = kcapi_kpp_setkey(key->handle, NULL, 0);
```

Comment:
This return code will return positive integer to indicate success. Please change line 123 to `if (ret >= 0) {`. Ref: http://www.chronox.de/libkcapi/html/API-kcapi-kpp-setkey.html

Reply:
fixed